setup.cfg

```diff
@@ -4,4 +4,5 @@ xfail_strict=true
 
 [flake8]
```

Comment:
We also have configuration in `.flake8`. From my reading of the docs I suspect that this file is taking precendent but I'm not 100% sure on that. https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations

Does this mean that we can get rid of the `.flake8` file?

Reply:
Removed `.flake8` in 0ed1f329